src/modules/rolle/repo/rolle.repo.ts

```diff
@@ -216,6 +216,39 @@ export class RolleRepo {
         }
     }
 
+    public async updateRolle(
```

Comment:
Can you formulate the checks made as a spec and just slap it into update?

Also: Is it possible to include update in the save method? The repo should be able to figure out whether to do update or save on its own. (Based on if we give it an ID or no

Reply:
Which checks? If you mean the check 'isRolleAlreadyAssigned' which is in the controller, we cannot move it in the RolleRepo because it creates a circular reference error because of the DBiamPersonenkontextRepo.

We have already a save method that does call create or save based on the ID of the aggregate. This new method updateRolle can be maybe renamed to 'updateRolleAuthorized' because it also checks the permissions & other things.